homeassistant/components/metoffice/weather.py

```diff
@@ -72,6 +77,11 @@ def _get_weather_condition(metoffice_code):
 class MetOfficeWeather(CoordinatorEntity, WeatherEntity):
     """Implementation of a Met Office weather condition."""
 
+    _attr_temperature_unit = TEMP_CELSIUS
+    _attr_pressure_unit = PRESSURE_HPA
+    _attr_wind_speed_unit = SPEED_MILES_PER_HOUR
+    _attr_precipitation_unit = LENGTH_MILLIMETERS
```

Comment:
MetOffice doesn't provide precipitation so this may be redundant

Reply:
Thanks! I've removed it.